<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Data_edit</class>
 <widget class="QDialog" name="Data_edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Мастер пол | Изменение данных</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>301</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Имя партнера</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_user_name_for_de"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Поле длря редактирования</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_user_data_change"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Ввод новые данных</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_user_data_Edit"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_save_data_edit">
      <property name="text">
       <string>Применить изменения</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
